package.json

```diff
@@ -29,15 +29,15 @@
     "commitplease": "2.3.1",
     "core-js": "2.2.2",
     "cross-spawn": "2.2.3",
+    "eslint-config-jquery": "github:markelog/eslint-config-jquery",
```

Comment:
> I'd like to avoid having commits on master that in some time will almost for sure stop compiling

For sure everything will stop working eventually i assure i do not plan to suddenly remove this repository :).

Didn't want to release `eslint-config-jquery` without conformation of code-style compliance. So i'm either switch to correct name right after this lands or even in here